linguatec_lexicon_frontend/templates/linguatec_lexicon_frontend/components/search-tools.html

```diff
@@ -2,6 +2,18 @@
     <div class="input-group rg-search">
         <label class="sr-only" for="input-search">Término a buscar</label>
         <input id="input-search" class="ui-widget form-control" name="q" type="text" value="{{ query }}">
+        <select name="l">
+            {% if lexicons %}
+                {% for lex_list_by_src in lexicons %}
+                    <optgroup label="{{ lex_list_by_src.0.src_language }}"> 
+                    {% for lex in lex_list_by_src %}
+                        <option value="{{ lex.name }}">{{ lex.name }}</option> 
+                    {% endfor %}
+                {% endfor %}
+            {% else %}
+                <option selected value=''>Empty Value</option>
+            {% endif %}
```

Comment:
Check the following docs and then refactor:
- https://docs.djangoproject.com/en/3.1/ref/templates/builtins/#for-empty

Reply:
Refeactored using the {% empty %} method of Django. Thank you, i didn't know that this {% empty %} existed